src/views/ContinuationIn/ContinuationInBusinessBc.vue

```diff
@@ -4,9 +4,13 @@
     <section class="mt-10">
       <header id="company-name">
         <h2>Company Name</h2>
+        <p></p>
```

Comment:
Don't use empty paragraphs for whitespace. Instead, use the Vuetify helper classes to set margins or padding. [link](https://v2.vuetifyjs.com/en/styles/spacing/)

Reply:
Got it, thanks. Updated.